PopPUNK/visualise.py

```diff
@@ -396,13 +396,12 @@ def generate_visualisations(query_db,
     isolateClustering = {}
     # Use external clustering if specified
     if external_clustering:
+        mode = 'external'
         cluster_file = external_clustering
-        isolateClustering = readIsolateTypeFromCsv(cluster_file,
-                                                   mode = 'external',
-                                                   return_dict = True)
-
+        suffix = "_clusters.csv"
+        if cluster_file.endswith('_lineages.csv'):
+            suffix = "_lineages.csv"
```

Comment:
An if/else is clearer:
```suggestion
        if cluster_file.endswith('_lineages.csv'):
            suffix = "_lineages.csv"
        else:
            suffix = "_clusters.csv"
```